nn-git-log/README.md

```diff
@@ -0,0 +1,113 @@
+```
+Thus when running the command.
+```bash
+git log --oneline
+```
+![Git log with only oneline](res/small_picture_2.png)
+
+And boom, this was what we wanted at the start. Now we can see that we have added the landing page and also changed the extension of the landing page.  
+
+But there is something weird here, we can also see that the `origin/master` commit together with the commits of our branch. This is because as we know, branches have to start off(be based off) somewhere. As such, we can make a pretty good guess that this `frontend/landing` was made off the master branch.  
+
+Luckily, we don't have to guess as there is a flag which will show us the relationship between these commits.  
+
+### Flag graph
+To make git log show the relationships between commits, we just have to add the `--graph` flag. This flag shows a 'graph' of the commits, which for us means the relationship the commits have.  
+Combining this with `--oneline` flag we showed above, makes for a very clean and concise git log.  
+
+(I will leave it to the reader to try out what `--graph` flag does on its own)
+
+```bash
+git log --oneline --graph
+```
+![Git log with oneline and graph](res/small_picture_3.png)
+
+## Looking at the big picture
+Now that you know how to look at commits tagged to your branch, you might also want to see how the entire repository might look like.  
+Luckily we have a flag to look at the log of the entire repository.  
+
+### Flag all
+Using the `--all` flag, this will show the git log, including all the commits from every branch you might have in the respository.  
+We will also be combining this flag with all previous flags to get the best overview of a project. (In my opnion)
+```bash
+git log --oneline --graph --all
+```
+![Git log with oneline graph and all](res/big_picture_1.png)
+
+Now we can see that our frontend landing page feature has already been merged down to master. We can also see that after git fetch(To fetch any changes from github), our teammate B had already made more commits on his feature and our local `backend/start-server` branch is behind.  
+
+## Wrapping up
+To end off, the `git log` command actually has a lot more features and flags which are a bit more nuanced and situationally. What was shown here are the most common and applicable features and flags which I hope will be usefully to you.
+
+As mentioned above, below are additional resources:  
+[More features of git log](https://www.atlassian.com/git/tutorials/git-log)
+[git log documentation](https://git-scm.com/docs/git-log)
+
+Additional GUI applications that can help you visualise the git log:  
+[VSCode extension (Git Graph)](https://marketplace.visualstudio.com/items?itemName=mhutchie.git-graph)
+![Git graph looks like](res/git_graph_vscode.png)
+
+On Github:
```

Comment:
Could be set to a subheading instead so it shows up when `pandoc` renders the PDF document for ease of reference

Reply:
Also not sure here, new to pandoc so I'm not so sure of how to set up so the conversion goes smoothly.
Tried referencing your previous readme although I'm not sure what you did.

Reply:
Just a regular subheading in markdown will do :+1: Will open another issue